contrib/bloop/test/src/mill/contrib/bloop/BloopTests.scala

```diff
@@ -80,6 +80,18 @@ object BloopTests extends TestSuite {
       // skipped on Windows
       val scalanativeModuleConfig = if(scala.util.Properties.isWin) None else Some(readBloopConf("scalanativeModule.json"))
 
+      "no-compilation" - {
+        val workspaceOut = os.pwd / "target" / "workspace" / "mill" / "contrib" / "bloop" / "BloopTests" / "testEvaluator"
+        val scalaModuleCompile = workspaceOut / "scalaModule" / "compile"
+        val scalaModule2Compile = workspaceOut / "scalaModule2" / "compile"
+
+        // Ensuring that bloop config generation didn't trigger compilation
+        assert(os.exists(workspaceOut / "scalaModule"))
+        assert(!os.exists(workspaceOut / "scalaModule" / "compile"))
+        assert(os.exists(workspaceOut / "scalaModule2"))
+        assert(!os.exists(workspaceOut / "scalaModule2" / "compile"))
```

Comment:
Are these `val`s unused? Should they be used in the assertions or removed?

Reply:
My bad, 2b60d12